frontend/src/lib/components/Support/supportLogic.ts

```diff
@@ -28,24 +29,24 @@ function getDjangoAdminLink(user: UserType | null): string {
     return `\nAdmin link: ${link} (Organization: '${user.organization?.name}'; Project: '${user.team?.name}')`
 }
 
-export const TargetAreaToName = {
-    analytics: 'Analytics',
+export const TARGET_AREA_TO_NAME = {
     app_performance: 'App Performance',
     apps: 'Apps',
+    login: 'Authentication (Login / Sign-up / Invites)',
     billing: 'Billing',
     cohorts: 'Cohorts',
-    data_management: 'Data Management',
     data_integrity: 'Data Integrity',
-    ingestion: 'Events Ingestion',
+    data_management: 'Data Management',
+    ingestion: 'Event Ingestion',
     experiments: 'Experiments',
     feature_flags: 'Feature Flags',
-    login: 'Login / Sign up / Invites',
-    session_replay: 'Session Replay',
+    analytics: 'Product Analytics (Insights)',
```

Comment:
Maybe:
```suggestion
    analytics: 'Product Analytics (Insights, dashboards, event explorer, annotations)',
```
You can also add them separately if you prefer making sure to update the url path mappings too. These were previously was covered under "Analytics" name kindof...

Reply:
Mostly added this just to disambiguate, as I think for some users we are a product analytics platform, so I've seen them select this even if their particular request is not really relevant to the product analytics area.